integration/spark2/src/main/scala/org/apache/carbondata/stream/StreamJobManager.scala

```diff
@@ -58,14 +58,16 @@ object StreamJobManager {
                                                 "streaming sink table " +
                                                 "('streaming' tblproperty is not 'sink' or 'true')")
     }
+    // TODO: validate query schema against sink ( as in kafka we cannot get schema directly)
+    /*
```

Comment:
can we move this validation before, so that we still do validation for non-kafka format?